includes/functions.php

```diff
@@ -63,6 +63,7 @@ function get_stories( array $attrs = [], array $query_args = [] ) {
  * @return void
  */
 function render_theme_stories() {
+	$customizer = new Customizer();
 	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
-	echo get_plugin_instance()->customizer->render_stories();
+	echo $customizer->render_stories();
```

Comment:
This could potentially be rewritten like this:
```php
echo Services::get( 'customizer' )->render_stories();
```

This avoids relying on `new` and thus keeps all indirection and dependency management intact.

Reply:
Note in case the static access to `Services::get()` looks problematic:
This is not ideal, but still better than `new Customizer()`. Using `new` is the same type of static access, only to `Customizer::__construct()` instead. However, what makes the solution with `new` worse is that you not only hard-coded a static access, but you also hard-coded the arguments for the constructor (so, how its dependencies are handled), and you hard-coded the way the instantiation is optimized and/or shared (i.e. not at all).

Reply:
As this is a function that is not even used in our codebase,I thought a new class is not really a problem here. I will convert to use the injector.

Reply:
fixed.